### PR TITLE
support dump workload bpf map

### PR DESCRIPTION
**Pull request by Okabe-Rintarou-0**
**What type of PR is this?**
feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by Okabe-Rintarou-0** [comment]
since we have some problems in ads bpf map deserialization, i first implemented the workload bpf map. 
cc @hzxuzhonghu

**Comment by Okabe-Rintarou-0** [comment]
WIP, exists some erros

**Comment by hzxuzhonghu** [comment]
Great. let me try

File: pkg/status/status_server.go [hunk 1]
```diff
@@ -257,7 +309,7 @@ func (s *Server) configDumpAds(w http.ResponseWriter, r *http.Request) {
 	client := s.xdsClient
 	if client == nil || client.AdsController == nil {
 		w.WriteHeader(http.StatusBadRequest)
-		fmt.Fprintf(w, "\t%s\n", "invalid ClientMode")
+		fmt.Fprint(w, invalidModeErrMessage)
```
Comment (YaoZengzeng) [inline pkg/status/status_server.go:312]: ditto

**Review by hzxuzhonghu — COMMENTED** [review]
Overall lgtm

File: pkg/status/status_server.go [hunk 2]
```diff
@@ -46,6 +47,7 @@ const (
 	patternHelp               = "/help"
 	patternOptions            = "/options"
 	patternBpfAdsMaps         = "/debug/bpf/ads"
+	patternBpfWorkloadMaps    = "/debug/bpf/workload"
```
[the remark this answers is not in the document]
Comment (Okabe-Rintarou-0) [reply pkg/status/status_server.go:50]: seems good

File: pkg/status/status_server.go (hunk 1 above)
Comment (Okabe-Rintarou-0) [reply pkg/status/status_server.go:312]: ok

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**